lib/util/Components.js

```diff
@@ -403,9 +404,11 @@ function componentRule(rule, context) {
               && requireExpression.callee
               && requireExpression.callee.name === 'require'
               && requireExpression.arguments[0]
-              && requireExpression.arguments[0].value === pragma.toLocaleLowerCase()
             ) {
-              return true;
+              pragmaLowerCase = pragmaLowerCase || pragma.toLocaleLowerCase();
```

Comment:
if you're looking for perf, why do an unconditional assignment?
```suggestion
              if (!pragmaLowerCase) { pragmaLowerCase = pragma.toLocaleLowerCase(); }
```

note as well that using `if` actually matches the semantics of `||=`.

Reply:
Good catch, I'll fix this.